r/fama-macbeth-regressions.qmd

```diff
@@ -33,6 +33,7 @@ The current chapter relies on this set of R packages.
 ```{r}
 #| message: false
 library(tidyverse)
+library(tidyfinance)
```

Comment:
I would not claim that the chapter relies on tidyfinance, we don't use the package here (eval = FALSE). I'd rather propose we remove calling library(tidyfinance) except for the chapters where we download data and bring it up for the "by the way, tidyfinance can do the same via the following"... part